### PR TITLE
Add Adafruit example.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: examples/adafruit.toit [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- create a feed: https://io.adafruit.com/floitsch_toit/feeds
+- update the $ADAFRUIT_IO_USERNAME, $ADAFRUIT_IO_KEY, and $ADAFRUIT_IO_FEEDNAME constants.
+*/
+
+import mqtt
+import net
+import certificate_roots
+
+ADAFRUIT_IO_USERNAME ::= "<YOUR_USERNAME>"
+// From io.adafruit.com/$ADAFRUIT_IO_USERNAME/dashboards -> click "My Key"
```
Comment (kasperl) [inline examples/adafruit.toit:22]: I'd add a newline before this comment.

File: examples/adafruit.toit [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  client := mqtt.Client --transport=transport
+
+  options := mqtt.SessionOptions
+    --client_id = "toit-example-client"
+    --username = ADAFRUIT_IO_USERNAME
+    --password = ADAFRUIT_IO_KEY
+
+  client.start --options=options
+
+  print "connected to broker"
```
Comment (kasperl) [inline examples/adafruit.toit:50]: I would either capitalize the string or use log.info.

**Review by kasperl — APPROVED** [review]
Looks good to me!

File: examples/adafruit.toit (hunk 1 above)
Comment (floitsch) [reply examples/adafruit.toit:23]: done.

File: examples/adafruit.toit (hunk 2 above)
Comment (floitsch) [reply examples/adafruit.toit]: done.

**Pull request merged**